src/models/event.ts

```
@@ -0,0 +1,77 @@
+      references: {
+        model: "users",
+        key: "id",
+      },
+    },
+    status: {
+      type: DataTypes.ENUM(...Object.values(EventStatus)),
+      allowNull: false,
+      defaultValue: EventStatus.DRAFT,
+    },
+    createdAt: "",
+    updatedAt: ""
```

ESLint

[error] Insert `,`

(prettier/prettier)